apps/website/src/content/docs/accessibility.mdx

```diff
@@ -0,0 +1,73 @@
+
+## Metadata
+
+Applications should set the [`lang` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) on the `<html>` element. This can be done from the application entrypoint (whether that's a plain HTML file or a JSX root layout). The value of the `lang` attribute should coincide with the current language of the interface.
+
+```html
+<html lang="en">
+  …
+</html>
+```
+
+Every page must have a unique `<title>` that identifies the page. For example, the application's home page might have the title "Home — Acme Application", and a projects page might have the title "Projects — Acme Application".
+
+```html
+<title>{pageName} — {applicationName}</title>
+```
+
+### Single-page applications
+
+When working with single-page applications, setting the title might require using your framework's built-in metadata API, or a third-party library like [`react-helmet`](https://github.com/nfl/react-helmet).
+
+When navigating between pages on a regular website, screen-readers will automatically announce the page title. Within a single-page application, you might need to use a [live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) to emulate the same functionality. Some frameworks come with a [built-in route announcer](https://nextjs.org/docs/architecture/accessibility#route-announcements), while others might require implementing this manually.
+
+In some cases, it may be necessary to programmatically move focus after a client-side navigation occurs. This helps ensure that the new content can be easily reached by keyboard users.
+
+## Landmarks
+
+[Landmarks](https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/) are a set of eight roles that identify the major sections of a page. Landmarks allow assistive technology users to quickly browse and skip to relevant content.
+
+The main content of the page must be wrapped in a [`<main>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/main) element. This works well when combined with iTwinUI's [`<Header>`](/docs/header), [`<SideNavigation>`](/docs/sidenavigation), and [`<Footer>`](/docs/footer) components.
+
+```jsx
+<Header>…</Header>
+<SideNavigation>…</SideNavigation>
+<main>…</main>
+<Footer />
+```
+
+iTwinUI provides an [iTwinUI-layouts package](https://itwin.github.io/iTwinUI-layouts/) which automatically sets up these landmarks, and also includes a [`<SkipToContentLink>`](/docs/skiptocontentlink) by default. Depending on the complexity of the content, you may need to set up additional landmarks for a better user experience.
```

Comment:
I will work on updating the layouts package so that this part is actually true.

@r100-stack @FlyersPh9 Can you review the rest of it in the meantime?

Reply:
I was looking into this, and realized that the claim "automatically sets up these landmarks" cannot be true, because iTwinUI-layouts does not actually render these landmarks. e.g. `PageLayout.Header` still requires passing `<Header>` as children.

There is also no "natural" place for us to render the `<main>` element, because `PageLayout.Content` and `PageLayout.BottomBar` (seen in [this demo](https://itwin.github.io/iTwinUI-layouts/#/page-layout-3)) can be siblings.

So I'm thinking of leaving out any mention of iTwinUI-layouts, and instead recommending adding `<SkipToContentLink>` manually.

Reply:
[Removed mention of itwinui-layouts](https://github.com/iTwin/iTwinUI/pull/2516/commits/96970e9df97de31ea3432b55bb0767ee77683860)